src/modules/navigation/AppRoute.jsx

```diff
@@ -89,8 +89,11 @@ const AppRoute = () => (
         >
           <Route path="v/revision" element={<FileHistory />} />
           <Route path="v/share" element={<ShareFileView />} />
-          <Route path="v/move" element={<MoveFilesView />} />
-          <Route path="v/duplicate" element={<FolderDuplicateView />} />
+          <Route path="v/move" element={<MoveFilesView isOpenInViewer />} />
+          <Route
```

Comment:
the change in this file is needed? i see it's a lint change but it's not the topic of this PR